sdk/core/Azure.Core.TestFramework/src/TestActivitySourceListener.cs

```diff
@@ -46,6 +48,13 @@ public TestActivitySourceListener(Func<ActivitySource, bool> sourceSelector)
             ActivitySource.AddActivityListener(_listener);
         }
 
+        public Activity AssertAndRemoveActivity(string name)
```

Comment:
nit: 
```suggestion
        public Activity RemoveActivityAndAssert(string name)
```

Reply:
This name actually matches the AssertAndRemoveScope method in ClientDiagnosticListener.